doc/guide/CircleListView.md

```diff
@@ -19,22 +18,21 @@ To receive [Rotary event](https://developer.tizen.org/development/training/nativ
 - [Cells](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/listview/customizing-cell-appearance#custom-cells) : Data in a `CircleListView` is presented in cells. Each cell corresponds to a row of data.
 - [Footer](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/listview/customizing-list-appearance#Headers_and_Footers) : Text or view that is displayed at end of a list.
 
-**WARNING: [CircleListView](xref:Tizen.Wearable.CircularUI.doc.CircleListView), [CircleDateTimeSelector](xref:Tizen.Wearable.CircularUI.doc.CircleDateTimeSelector), [CircleScrollView](xref:Tizen.Wearable.CircularUI.doc.CircleScrollView), [CircleStepper](xref:Tizen.Wearable.CircularUI.doc.CircleStepper) must be confined in the `CirclePage` container or [Page](https://developer.xamarin.com/api/type/Xamarin.Forms.Page/) with [CircleSurfaceEffectBehavior](xref:Tizen.Wearable.CircularUI.doc.CircleSurfaceEffectBehavior). If you add these controls in any other way,  it may cause an exception or cannot display the controls.**
+## Create CircleListView
 
-## Add CircleListView in CirclePage
+Basically `CircleListView` looks same as [Xamarin.Forms.ListView](https://developer.xamarin.com/api/type/Xamarin.Forms.ListView/).
+The difference from [Xamarin.Forms.ListView](https://developer.xamarin.com/api/type/Xamarin.Forms.ListView/) is to provide some additional property for Tizen wearable such as `BarColor`
 
-You can set `CircleListView` in the [CirclePage.Content](xref:Tizen.Wearable.CircularUI.doc.CirclePage). For more information on how to add a [CirclePage](xref:Tizen.Wearable.CircularUI.doc.CirclePage), see [CirclePage guide](https://samsung.github.io/Tizen.CircularUI/guide/CirclePage.html#create-circlepage).
-`RotaryFocusObject` property sets the currently focused control using the rotating interaction, and displays the focused control's circle object.
-If the value is not set properly,  the control will not receive the [Rotary Event](https://developer.tizen.org/development/training/native-application/understanding-tizen-programming/event-handling#rotary).
+`CircleListView` has the following property:
 
-You can either set the `Header` and/or `Footer` to a simple text, or to a more complex layout. Using `HeaderTemplate` and `FooterTemplate` properties you can create more complex layouts for the header and footer that support data binding.
+- BarColor: This property gets or sets a scroll bar color value.
 
+You can either set the `Header` and/or `Footer` to a simple text, or to a more complex layout. Using `HeaderTemplate` and `FooterTemplate` properties you can create more complex layouts for the header and footer that support data binding.
```

Comment:
You can either set the `Header` and/or `Footer` with the simple text or with a more complex layout. Using `HeaderTemplate` and `FooterTemplate` properties, you can create more complex layouts for the header and footer including data binding.